### DISCUSSION TITLE
How to reinstall older version of idea

### DISCUSSION THREAD

**Discussion by wiatrgr** [ScoopInstaller/Scoop#6152] [Help and Questions]
I have updated idea app to the newest version but its not working, apparently there is a bug, missing api-ms-win-core-console-l1-2-0.dll .

I tried to downgrade to version 2024.1, but its not working. How to fix it ? I wanted to install with: 

scoop install idea@2024.1.5-241.18968.26 but I got error: 

PS C:\Users\wiatrgr> scoop install idea@2024.1.5-241.18968.26 -g
WARN  Given version (2024.1.5-241.18968.26) does not match manifest (2024.2.2-242.22855.74)
WARN  Attempting to generate manifest for 'idea' (2024.1.5-241.18968.26)
Autoupdating idea
Searching hash for ideaIC-$matchVer.win.zip in https://download.jetbrains.com/idea/ideaIC-$matchVer.win.zip.sha256
The remote server returned an error: (404) Not Found.
URL https://download.jetbrains.com/idea/ideaIC-$matchVer.win.zip.sha256 is not valid
Could not find hash in https://download.jetbrains.com/idea/ideaIC-$matchVer.win.zip.sha256
Downloading ideaIC-$matchVer.win.zip to compute hashes!
The remote server returned an error: (404) Not Found.
URL https://download.jetbrains.com/idea/ideaIC-$matchVer.win.zip is not valid
Could not install idea@2024.1.5-241.18968.26
Select-CurrentVersion : Cannot bind argument to parameter 'AppName' because it is an empty string.

**Comment by HUMORCE** [comment]
The given version is available, please check your internet connection.

```
❯ scoop download idea@2024.1.5-241.18968.26
INFO  Downloading 'idea' (2024.1.5-241.18968.26) [64bit] from extras bucket
WARN  Given version (2024.1.5-241.18968.26) does not match manifest (2024.2.2-242.22855.74)
WARN  Attempting to generate manifest for 'idea' (2024.1.5-241.18968.26)
ideaIC-2024.1.5.win.zip (802.2 MB) [======================================================================================] 100%
Checking hash of ideaIC-2024.1.5.win.zip ... ok.
'idea' (2024.1.5-241.18968.26) was downloaded successfully!
```
